<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Chatting</class>
 <widget class="QMainWindow" name="Chatting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>508</width>
    <height>454</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Chatting</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>471</width>
      <height>401</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>닉네임 : </string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLineEdit" name="nicname"/>
     </item>
     <item row="3" column="0" colspan="3">
      <widget class="QListWidget" name="listWidget"/>
     </item>
     <item row="0" column="2">
      <widget class="QLineEdit" name="ipLineEdit"/>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>파일전송 IP :</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>포트 : </string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>메시지 :</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="portLineEdit"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>IP 주소 :</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3" rowspan="3">
      <widget class="QPushButton" name="connectButton">
       <property name="text">
        <string>접속</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2" colspan="2">
      <widget class="QLineEdit" name="messageLineEdit"/>
     </item>
     <item row="5" column="0" colspan="4">
      <widget class="QPushButton" name="sendButton">
       <property name="text">
        <string>전송</string>
       </property>
      </widget>
     </item>
     <item row="7" column="2" colspan="2">
      <widget class="QLineEdit" name="ipfilesendEdit"/>
     </item>
     <item row="8" column="0" colspan="4">
      <widget class="QPushButton" name="fileSendButton">
       <property name="text">
        <string>파일 전송</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
